test/collectionUtils/testImpls/ExtractValueTransformer.ts

```diff
@@ -0,0 +1,17 @@
+import Transformer from "../../../src/collectionUtils/core/Transformer";
+
+export interface IValue {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)